pagopa/io-react-native-proximity
# [SIW-777,SIW-775] Add check to start proximity manager only once and removeListeners method

### This PR depends on #16 

### Short description

This PR updates start method of proximity manager adding a boolean value to track start status. Start method it should be called only once otherwise multiple connection could be happened causing duplication of buffered data. This PR also adds a method to remove all listeners.

#### List of Changes
- Update index.ts
- Update EventManager
- Update example app

#### How Has This Been Tested?

- Start the example app
- Tap on Start button
- Generate QR Code
- Scan the with a verifier app (after verification the app should stop the proximity manager automatically)
- Tap again on Start button and retry proximity flow
